scss/global/_typography.scss

```diff
@@ -0,0 +1,9 @@
+h2 {
+  font-size: 32px;
+  color: $color-white;
+  line-height: 1.2;
+  text-shadow: 0px 2px 2px $color-temptress;
```

Comment:
jak masz jakas wartosc 0 to nie musisz dawac px, wystarczy samo 0

Reply:
Done